cmd/vdev_id/vdev_id

```diff
@@ -186,7 +186,7 @@ sas_handler() {
 
 		# Get the raw scsi device name from multipath -l.  Strip off
 		# leading pipe symbols to make field numbering consistent.
-		DEV=`multipath -l $DM_NAME |
+		DEV=`multipath -ll $DM_NAME |
```

Comment:
Can you update the comment above this line with a sentence indicating why `multipath -ll` is being used?